.lintr

```diff
@@ -0,0 +1,12 @@
+linters: with_defaults(
+  line_length_linter(120), 
+  commented_code_linter = NULL,
+  object_name_linter = object_name_linter("camelCase"),
+  open_curly_linter = NULL,
+  closed_curly_linter = NULL,
```

Comment:
I don't think these need to be disabled. I don't see any linter errors with them enabled and they enforce a good chunk of styler's rules.

Reply:
Unfortunately, `styler` wants to put the curly braces on a newline, and `lintr` doesn't. I'd rather have configuration for only `lintr`, and just keep `styler` formatting set to the defaults, so I think we need these. GH issue for reference:
https://github.com/r-lib/styler/issues/438

Reply:
Oh weird. I only looked at our R/ files, but I see the test files cause styler/lintr battles. Styler is correct, lintr is wrong (https://github.com/jimhester/lintr/issues/487). Adding `open_curly_linter(allow_single_line=TRUE)` doesn't seem to work.